ui/pages/Project/reducers.js

```diff
@@ -370,7 +371,15 @@ export const updateFamiliesTable = (updates, tableName) => (
   { type: tableName === CASE_REVIEW_TABLE_NAME ? UPDATE_CASE_REVIEW_TABLE_STATE : UPDATE_FAMILY_TABLE_STATE, updates }
 )
 
-export const updateFamiliesTableFilters = updates => ({ type: UPDATE_FAMILY_TABLE_FILTER_STATE, updates })
+export const updateFamiliesTableFilters = updates => (dispatch, getState) => {
```

Comment:
Because the data structure is changing, you need to change what kind of reducer `UPDATE_FAMILY_TABLE_FILTER_STATE` is, so its one that can be easily partially updated. If you change it to `createObjectsByIdReducer` then the built in update functionality will handle merging old and new state for you, and you wouldn't need `getFamiliesTableFiltersByProject` at all here